routes/feedback.py

```diff
@@ -28,5 +28,29 @@ async def delete_event(event_id, feedback_id):
 
     # prevent console error
     return Response(status_code=204) # https://github.com/tiangolo/fastapi/issues/717#issuecomment-583826657
+"""???doesnt need endpoint???
+@router.add_upvote{
+    "/feedback/???/{event_id}/{feedback_id}",
+    #description=
+    #summary=
+    #tags=
+    status_code=200,
+
+}
+"""
+async def add_upvote_to_feedback(event_id, feedback_id):                   
+
+    # get DB Database
+    db = get_database()
+
+    # increase upvote count
+    await utils.add_upvote_to_feedback(feedback_id, db)
```

Comment:
we don't actually need event_id here, or anywhere (we're not querying into the event database at all just to update the feedback object).